docs/specs/toc.yml

```diff
@@ -600,7 +600,7 @@ outputs:
 # replace href finally
 inputs:
   docfx.yml: |
-    files: docs/f1/TOC.yml
+    files: docs/**/TOC.*
```

Comment:
what if we keep the test unchanged, do we changed the behavior?

Reply:
in this test case, `TOC.md` is resolved using dynamic scope (which is unexpected). We only do dynamic scope for resources.

Reply:
if it's belongs to inclusion, i think the behavior is just like token, right?

Reply:
This is a weird case, and feels more like a link.
It does not embed the content of the sub TOC, it just use the first link in sub TOC.

Reply:
Either way is OK. And if it is treated like inclusion, it will also handle dynamic scope, dependency map, git history fallback for loc, etc.